fhir-search/src/main/java/com/ibm/fhir/search/compartment/CompartmentHelper.java

```diff
@@ -154,14 +154,23 @@ public static final void buildMaps(Map<String, CompartmentCache> compMap, Map<St
         return compartmentMap.get(compartment).getParametersByResourceTypeInCompartment(resourceType);
     }
 
+    /**
+     * checks that the compartment is valid and throws an exception if not
```

Comment:
This mentions an exception, but this method doesn't throw one.

Reply:
oops, i removed the `@exception` param but missed that it was in the description too.  fixed